web/templates/status.html

```diff
@@ -56,7 +56,9 @@
                 </span>
               </td>
               <td>
-                {{stripLabels .BaseLabels "job" "instance"}}
+                <a href="#" data-toggle="tooltip" title="" data-original-title="Before Relabel: {{.MetaLabels}}">
```

Comment:
Yeah, this should go on a separate page but there was a fair bit of plumbing involved for that to work so I went with something simple to start with.